sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -774,13 +774,42 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder {
   }
 
   /**
-   * Create an [[AddJarCommand]] or [[AddFileCommand]] command depending on the requested resource.
+   * Create a [[AddFileCommand]], [[AddJarCommand]], [[ListFilesCommand]] or [[ListJarsCommand]]
+   * command depending on the requested operation on resources.
+   * Expected format:
+   * {{{
+   *   ADD (FILE[s] [filepath ...] | JAR[s] [jarpath ...])
+   *   DELETE (FILE[s] [filepath ...] | JAR[s] [jarpath ...])
+   *   LIST (FILE[s] [filepath ...] | JAR[s] [jarpath ...])
```

Comment:
`[S]` instead of `[s]`? Also, arguments like `filepath` and `jarpath` are required, and should be written in angle brackets (`<filepath ...>` and `<jarpath ...>`).